front/app/modules/commercial/content_builder/admin/components/CraftComponents/Image/index.tsx

```diff
@@ -21,6 +21,10 @@ import messages from '../../../messages';
 // intl
 import { injectIntl } from 'utils/cl-intl';
 
+// events
+import eventEmitter from 'utils/eventEmitter';
+import { IMAGE_UPLOADING_EVENT } from '../../../containers';
```

Comment:
Non-blocking, we could use the suggested approach in last week's meeting for the new imports.

`import { IMAGE_UPLOADING_EVENT } from 'modules/commercial/content_builder/admin/containers';`